application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java

```diff
@@ -736,7 +736,7 @@ private static MeasureType getMeasureType(int value, String unitCode, String lis
         faCatch_1[1] = "BEAGLE";
         faCatch_2[1] = "SEAFOOD";
         faCatch_3[1] = "BEAGLE";
-        faCatch_4[0] = "BEAGLE";
+        faCatch_4[1] = "BEAGLE";
```

Comment:
It disturbs me that this doesn't make a difference anywhere else

Reply:
I agree, not very thorough tests...